src/Sql/Sql/ManagedInstance/Cmdlet/NewAzureSqlManagedInstance.cs

```diff
@@ -305,6 +305,15 @@ public class NewAzureSqlManagedInstance : ManagedInstanceCmdletBase
         [PSArgumentCompleter("None", "1.0", "1.1", "1.2")]
         public string MinimalTlsVersion { get; set; }
 
+        /// <summary>
+        /// Gets or sets the managed instance backup storage redundancy
+        /// </summary>
+        [Parameter(Mandatory = false,
+            HelpMessage = "The Backup storage redundancy used to store backups for the Sql Azure Managed Instance. Options are: None, LRS, ZRS and RA-GRS ")]
+        [ValidateSet("LRS", "ZRS", "RA-GRS")]
+        [PSArgumentCompleter("LRS", "ZRS", "RA-GRS")]
```

Comment:
Powershell will supply argument completer when the parameter has a validate set automatically and so you can remove PSArgumentCompleter here.